pkg/cmd/server/origin/master.go

```diff
@@ -582,6 +573,23 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		"clusterRoles":          clusterRoleStorage,
 	}
 
+	if sdnovs.IsOpenShiftNetworkPlugin(c.Options.NetworkConfig.NetworkPluginName) {
```

Comment:
You can't do this - it's a breaking change when cluster config changes, and we don't do it.  All storage should always be enabled.